### PR TITLE
Update Beta information on IP protection

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
@MarkvanMents , David reached out to me asking me to update information on Private Beta in IP Protection documents to the limited access information we have in the refguide: https://docs.mendix.com/refguide/module-settings/.
Afaik, you were working on IP Protection, could you review my changes and leave your comments if any?

**Comment by Adam-Dupaski** [comment]
@MariaShaposhnikova we need to clarify ownership of these docs here:
-Solutions: me
-Connectors: @johannahemminger 

@johannahemminger can you take a look at these edits for your Connector doc?

**Comment by Adam-Dupaski** [comment]
@MariaShaposhnikova so David wants to call it "limited" and not "Private Beta"? Did he give a reason?

**Comment by Adam-Dupaski** [comment]
@MariaShaposhnikova  Because we do provide "Private Beta" as an option here: https://docs.mendix.com/releasenotes/beta-features/

But I guess it may make sense since the Vendor Program is not a Private Beta thing, it's a specific (limited) thing?

File: content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md [hunk 1]
```diff
@@ -188,7 +189,7 @@ Any dependencies your module has should be well documented including the minimum
 
 ### 3.9 IP Protection
 
-Right now, you will not be able to protect your IP or prevent end-users from changing any logic that you ship in a module using Mendix tooling. This functionality is currently in Private Beta (see [Applying IP Protection](/appstore/creating-content/sol-ip-protection/) for more information).
+Right now, you will not be able to protect your IP or prevent end-users from changing any logic that you ship in a module using Mendix tooling. Access to this functionality is currently limited and can be gained through [Mendix Vendor Program](/appstore/creating-content/vendor-program/).
```
Comment (johannahemminger) [inline content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md]: @MariaShaposhnikova I think we need to add "the" in front of Mendix Vendor Program, can I update that in both docs or will you also do for yours?

File: content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ We recommend that the *main module* for your connector include the following:
         * As an alternative to the version constant, you can place a subfolder with the version indication
 * **Private** (folder)– indicates what other developers should not touch when implementing your module by placing all of that logic in this folder
 * **UseMe** (folder) – contains everything the implementing developer could use to implement your module in their application, and might include subfolders for **Microflows**, **Pages**, **Snippets**, **Templates**, and **Constants**
-    
+
```
Comment (johannahemminger) [inline content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md:40]: @MariaShaposhnikova I will remove the extra line, unless there is a reason to keep it in there?

**Comment by MariaShaposhnikova** [comment]
@Adam-Dupaski , we discussed this with David some time ago when I prepared drafts for add-on modules and solutions. As I understand, it is the description of Beta limitations that is not applicable to them, they are not in Beta, but the access is limited. I did not know that the same should apply to IP Protection, but it makes sense because IP Protection can be enabled in add-on and solution modules.

File: content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/appstore/creating-content/connector-guide-build/connector-guide-best-practices.md]: Thanks @johannahemminger! I've updated it in all documents.

**Comment by MariaShaposhnikova** [comment]
@Adam-Dupaski and @johannahemminger , if you have no other comments, I will merge the PR. David has seen it and said it looked good.

**Pull request merged**